ols/app/endpoints/ols.py

```diff
@@ -96,7 +97,13 @@ def conversation_request(llm_request: LLMRequest) -> LLMResponse:
                 # See the comment in `summarize` method in `docs_summarizer.py`
                 # Because of that, we are ignoring some type checks when we
                 # are creating response model.
-                response = llm_response.response
+                if isinstance(llm_response, str):
```

Comment:
we should not need this.  if docs_summarizer.summarize isn't consistently returning a single type, please fix it, don't work around it like this.

Reply:
@bparees yes - it is happening because we are trying to support both llama_index and langchain